puppet/zulip/files/nginx/zulip-include-frontend/app

```diff
@@ -43,7 +43,7 @@ location /api/v1/events {
 }
 
 # Handle X-Accel-Redirect from Tornado to Tornado
-location ~ ^/tornado/(\d+)(/.*)$ {
+location ~ ^/internal/tornado/(\d+)(/.*)$ {
```

Comment:
I guess this redirect isn't used yet in Zulip Cloud (since it's part of a sharding feature we don't need there), so we don't need to be careful about compatibility for this detail during the cutover during an upgrade.

Reply:
Yeah, there are no deployment considerations with this unless you do per-user sharding and deploy with skip-puppet.